priv/repo/seeds.exs

```diff
@@ -1,11 +1,29 @@
 alias ElixirBench.{Benchmarks, Repos}
 
+github_client = Application.get_env(:elixir_bench, :github_client)
+
+if(github_client != ElixirBench.Github.ClientInMemory) do
+  raise """
+  \n
+  #############################################################################
+  Please, before running this seed you need to set :github_client to
+  ElixirBench.Github.ClientInMemory in your config/mix.#{Mix.env()}
+
+  This is necessary because our current scripts are not yet stable enough, so
+  you need to use local calls to mocked data instead of reaching the Github
+  servers in order to fetch the config.yml configuration for the repositories.
```

Comment:
why are they not stable enough? What's the problem? Can we fix it? Doing this seems a bit weird.

Reply:
The jobs used in the seeds are references for the outdated branch pushed by Michal. They miss some configurations that we've added, the `wait` option to be more precise. And as the code is not on `ecto` master yet, I would have to reference my forked repository with the correct configs and I don't think this is appropriate. Taking them from `Github.ClientInMemory` also ensure that future changes will not break as we use it in our tests.

Another option is to change the `config.dev.exs` to use the `Github.ClientInMemory` by default. Wdyt?

Reply:
Perhaps I used the wrong words in the message

Reply:
Nah I understand, having seeds not dependent on Internet connectivity is also a much smarter choice. Thanks for explaining! :+1: